resources/sass/_typography.scss

```diff
@@ -1,46 +1,39 @@
-@mixin link {
-    color: $font-color-progressive;
-    text-decoration: none;
-
-    &:focus-visible { // for chrome
-        outline: dotted;
-        outline-width: 1px;
-        outline-color: gray;
-    }
-
-    &:hover {
-        text-decoration: underline;
-    }
+@mixin body-M {
+    font-family: $font-family-system-sans;
+    font-size: $font-size-medium;
+    font-weight: $font-weight-normal;
+    line-height: $line-height-medium;
+    color: $color-base;
 }
 
-@mixin body-text {
-    font-family: $font-family-style-body;
-    font-size: $font-size-style-body;
-    font-weight: $font-weight-style-body;
-    line-height: $font-line-height-style-body;
-    color: $font-color-base;
+@mixin body-S {
+    font-family: $font-family-system-sans;
+    font-size: $font-size-small;
+    font-weight: $font-weight-normal;
+    line-height: $line-height-medium;
```

Comment:
This is possibly a mistake and should be `$line-height-small`

Reply:
the token is correct